### DISCUSSION TITLE
Android application crashes after successful building.

### DISCUSSION THREAD

**Discussion by jeffinhk** [software-mansion/react-native-screens#1630] [Q&A] [answered]
As the title suggests, my Android application is crashing the application after Metro successfully builds the project.
It shows no error whatsoever, and continuously crashes the app in splash screen stage.
I was working perfectly fine until few hours ago and this happened all the sudden.

I have tried upgrading the gradle version
I have tried cleaning the cache from Metro, gradlew, and even the computer itself.

I don't know how to start issuing this problem or show the crash log, so help me out and feel free to require me the information you need to reply my problem here.

What can I be missing?

**Comment by kkafar** [comment, marked as answer]
Hi @jeffinhk,

I guess your issues were related to [this recent issue from React Native](https://github.com/facebook/react-native/issues/35210). Everything should work just fine now I believe. If you are interested in what actually happened you can find excellent explanation under the link i provided.

**Reply by jeffinhk** [reply]
My problem has been solved(?) by upgrading to 0.70.1.
It took few days to rebuild my codes in new version, but I think you were right!
Thank you for your help.
